### ISSUE TITLE
Conflict when install using @mantine/core@6.0.13

### ISSUE THREAD

**Issue by zgramming** [icflorescu/mantine-datatable#349]
**Repository**
https://github.com/zgramming/Nextine

**To Reproduce**
type in terminal `npm i @mantine/core @mantine/hooks @emotion/react mantine-datatable`

**Expected behavior**
Install without problem

**Log**
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: nextine@0.1.0
npm ERR! Found: @mantine/core@6.0.13
npm ERR! node_modules/@mantine/core
npm ERR!   peer @mantine/core@"6.0.13" from @mantine/dates@6.0.13
npm ERR!   node_modules/@mantine/dates
npm ERR!     @mantine/dates@"^6.0.13" from the root project
npm ERR!   peer @mantine/core@"6.0.13" from @mantine/modals@6.0.13
npm ERR!   node_modules/@mantine/modals
npm ERR!     @mantine/modals@"^6.0.13" from the root project
npm ERR!   4 more (@mantine/notifications, @mantine/nprogress, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! mantine-datatable@"*" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @mantine/core@6.0.14
npm ERR! node_modules/@mantine/core
npm ERR!   peer @mantine/core@"^6.0.14" from mantine-datatable@2.6.4
npm ERR!   node_modules/mantine-datatable
npm ERR!     mantine-datatable@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!   4 more (@mantine/notifications, @mantine/nprogress, ...)
npm ERR! Could not resolve dependency:
npm ERR! @mantine/core@"^6.0.13" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @mantine/hooks@6.0.14
npm ERR! node_modules/@mantine/hooks
npm ERR!   peer @mantine/hooks@"6.0.14" from @mantine/core@6.0.14
npm ERR!   node_modules/@mantine/core
npm ERR!     @mantine/core@"^6.0.13" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\zeffr\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\zeffr\AppData\Local\npm-cache\_logs\2023-06-26T15_39_44_504Z-debug-0.log
```

**Desktop (please complete the following information):**

- OS: Windows

**Comment by icflorescu** [comment]
Thanks for bringing this to my attention.
I guess I was too strict with `peerDependencies`.

This should be fixed in a few minutes, as soon as `v2.6.5` hits the npm registry.

**Comment by zgramming** [comment]
@icflorescu Thank you for your fast response 🚀
